ts/src/config.ts

```diff
@@ -77,9 +78,28 @@ export interface Config extends AuthenticationConfig {
   // stack depth may increase overhead of profiling.
   heapMaxStackDepth?: number;
 
-  // Time to wait before trying to create a profile again if profile creation
-  // fails.
-  backoffMillis?: number;
+  // On each consecutive error in profile creation, the backoff envelope will
+  // increase by this factor. The backoff will be a random value selected
+  // from a uniform distribution between 0 and the backoff envelope.
+  backoffMultiplier?: number;
+
+  // On first error during profile creation, if the backoff is not specified
+  // by the server response, then profiler will wait between 0 and
+  // initialBackoffMillis before asking the server to create a profile again.
+  // After a successful profile creation, the backoff envelope will be reset to
+  // initialExpBackoffMillis.
+  initialBackoffMillis?: number;
+
+  // If the backoff is not specified by the server response, then profiler will
+  // wait at most expBackoffMillisCap before asking server to create a profile
+  // again.
+  expBackoffMillisCap?: number;
```

Comment:
errorBackoffCapMillis: units should go last, "exp" is not the purpose, handling errors is the purpose.

Reply:
I would actually name it just backoffCapMillis to make consistent with backoffMultiplier and initialBackoffMillis which do not carry the prefix. So, would only use the "server" prefix for configuration related to the server instructed backoff.

Reply:
Done.